### ISSUE TITLE
How do I make it run on Mac?

### ISSUE THREAD

**Issue by MonsieurSingh** [rohilpatel1/iTunes-Discord-Rich-Presense#5]
I installed the Macos-x64.rar file but it keeps on giving me this error:

<img width="372" alt="image" src="https://user-images.githubusercontent.com/66196010/120420336-8730a900-c3a7-11eb-8f7d-1e5ca943d9ee.png">

**Comment by rohilpatel1** [comment]
Sorry about that; I'll take a look!

**Comment by MonsieurSingh** [comment]
Did you find anything?

**Comment by rohilpatel1** [comment]
> Did you find anything?

Yep we did all right!! Sorry for the huge delay but we will be rolling out our second generation of this product any time soon, so make sure to follow me or frequently check my profile for it!

The project was too buggy to maintain so we decided to come out with a completely new release, so its on my profile. Sorry about that!!

**Comment by rohilpatel1** [comment]
> Did you find anything?

https://github.com/rohilpatel1/applemusicrpc Is the new release. Try this
